nrf_security/src/mbedtls/oberon/CMakeLists.txt

```diff
@@ -135,3 +135,14 @@ configure_file_ifdef(CONFIG_OBERON_MBEDTLS_ECJPAKE_C
   ${NRF_OBERON_MBEDTLS_INCLUDE_PATH}/ecjpake_alt.h
   ${generated_include_path}/ecjpake_alt.h
   COPYONLY)
+
+configure_file_ifdef(CONFIG_OBERON_MBEDTLS_CHACHA20_C
+  ${NRF_OBERON_MBEDTLS_INCLUDE_PATH}/chacha20_alt.h
+  ${generated_include_path}/chacha20_alt.h
+  COPYONLY)
+
+configure_file_ifdef(CONFIG_OBERON_MBEDTLS_POLY1305_C
+  ${NRF_OBERON_MBEDTLS_INCLUDE_PATH}/poly1305_alt.h
+  ${generated_include_path}/poly1305_alt.h
+  COPYONLY)
+  
```

Comment:
Why the extra line, and then a no newline:
![image](https://user-images.githubusercontent.com/9802583/95452854-311b7b00-096a-11eb-9808-b8989d3bb25d.png)

Reply:
fixed